README.md

```diff
@@ -6,13 +6,49 @@
 Cross-platform test runner written for Android and iOS projects
 
 ## Main focus
-- **stability** of test execution adjusting for flakiness in the environment and in the tests. 
+- **stability** of test execution adjusting for flakiness in the environment and in the tests.
 - **performance** using high parallelization (handling dozens of devices)
 
 ## Documentation
 
 Please check the official [documentation](https://marathonlabs.github.io/marathon/) for installation, configuration and more
 
+## [iOS Only] Added Support to consume tags for test functions under XCTestCase Subclasses
```

Comment:
Please move the docs info to the https://github.com/MarathonLabs/marathon/blob/develop/docs/_posts/2018-11-19-ios.md. They're automatically published at https://marathonlabs.github.io/marathon/